Java/E-montazysta/src/main/java/com/emontazysta/model/AppUser.java

```diff
@@ -29,14 +26,30 @@ public class AppUser implements UserDetails {
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
+    @NotNull
+    @NotEmpty
```

Comment:
@NotEmpty ma już w sobie sprawdzenie na not null, więc @NotNull jest w tym przypadku niepotrzebne
@NotEmpty zamieniłbym na @NotBlank które sprawdza również czy nie są wstawione same puste znaki.

Reply:
Tak, jak najbardziej mozna zmienić w sumie to tak bardziej przykładowo chciałem pokazać, że rózne są te adnotacje